app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] As part of this module requirement, _Post_ model should have a method that returns the **5 most recent comment**s for a given post. This should be similar to:

```ruby
# app/models/post.rb
...

def recent_comments
  comments.order('created_at Desc').limit(5)
end